src/mod_dirlisting.c

```diff
@@ -87,6 +87,7 @@ static excludes_buffer *excludes_buffer_init(void) {
 	return exb;
 }
 
```

Comment:
Incorrect (or at least misleading).  This looks to be part of warnings generated if lighttpd is built ./configure --without-pcre and should be addressed separately.